README.md

```diff
@@ -148,17 +150,42 @@ kubectl create -n <ns> -f config/samples/cluster_v1beta1_backup.yaml
 
 # Testing
 
-Example of a Backup execution with a backup in progress
+## Backup resources
+
+After you create a `backup.cluster.open-cluster-management.io` resource you should be able to run `oc get cbkp -A` and get the status of the backup executions.
+
+In the example below, the last completed backup is `backup-acm-2021-08-10-140404` and a new backup is currently running `backup-acm-2021-08-10-151345`.
+The name of the backup follows this template: `acm-backup-<timestamp>`
+
+Example of a `backup.cluster.open-cluster-management.io` execution with a `velero.io.backup` in progress:
 
 ```
 $ oc get cbkp -A
```

Comment:
I know this is not part of your PR but may we change this command? `oc get cbkup -n oadp-operator`

Reply:
changed to `oc get cbkup -n oadp-operator`
I meant to discuss this with you Dario; we can work on it in a separate task though
The issue : we need to create our own resources in the same ns with the velero resources - because we want to own the Schedule resource we create. But then there is no point in asking for veleroConfigBackupProxy since that is the ns in which our resource was created, so we have that info already
```
spec:
  veleroConfigBackupProxy: 
    metadata: oadp-operator
```